apis/beacon/attestations/rewards.yaml

```diff
@@ -0,0 +1,60 @@
+get:
+  operationId: getAttestationsRewards
+  summary: Get attestations rewards
+  description: Retrieves rewards info for attestations for validators specified in id. If none is specified, return rewards info for every validators.
+  tags:
+    - Beacon
+    - Rewards
+  parameters:
+    - name: epoch
+      in: path
+      required: true
+      description: "The epoch to get rewards info from"
+      schema:
+        $ref: "../../../beacon-node-oapi.yaml#/components/schemas/Uint64"
+    - name: id
+      description: "Either hex encoded public key (any bytes48 with 0x prefix) or validator index"
+      in: query
+      required: false
+      schema:
+        type: array
+        maxItems: 30
+        uniqueItems: true
+        items:
+          description: "Either hex encoded public key (any bytes48 with 0x prefix) or validator index"
+          type: string
+  responses:
+    "200":
+      description: "Success"
+      content:
+        application/json:
+          schema:
+            title: GetAttestationsRewardsResponse
+            type: object
+            properties:
+              data:
+                $ref: '../../../beacon-node-oapi.yaml#/components/schemas/AttestationsRewards'
```

Comment:
As well as the `data` we should also return `execution_optimistic` and `finalized` properties. The `finalized` one was just added recently in https://github.com/ethereum/beacon-APIs/pull/254.